.travis.yml

```diff
@@ -8,7 +8,7 @@ install:
   - "pip install -r requirements.txt"
   - "pip install -r tests/requirements.txt"
   - "pip install pytest-cov coveralls"
-  - "pip install git+https://github.com/projectatomic/osbs-client"
+  - "pip install git+https://github.com/projectatomic/osbs-client@ensure-image-stream-tag"
```

Comment:
This needs to be reverted before merge. It was added simply to show that unit tests are working as expected until `ensure-image-stream-tag` is merged into `osbs-client`.